tests/test_cookiecutter_generation.py

```diff
@@ -84,3 +84,14 @@ def test_flake8_compliance(cookies):
         sh.flake8(str(result.project))
     except sh.ErrorReturnCode as e:
         pytest.fail(e)
+
+
+def test_connection_max_age(cookies):
+    result = cookies.bake()
+
+    prod_config_file = result.project.join('config', 'settings', 'production.py')
+    prod_config_lines = prod_config_file.read()
+
+    expected = "DATABASES['default']['CONN_MAX_AGE'] = env.int('CONN_MAX_AGE', default=60)"
+
+    assert expected in prod_config_lines
```

Comment:
@browniebroke I'm not sure what do we need this test for?..

Reply:
I've used it to verify that my template variable was rendered properly.

Reply:
I have mixed feelings about that test: Should the default value change the test will also fail albeit altering the default value is a _non-functional_ change.

Reply:
I don't have any strong opinions. Any thoughts from the other maintainers?
Who has the final word on this? I'll remove it if no one else object.

Reply:
@pydanny, @jayfk, @luzfcb what do you think?

Reply:
LGTM